### PR TITLE
Changes required for version working within an AWX context

### PR DESCRIPTION
**Pull request by hollysilk**
Changes required for version working within an AWX context

### CODE REVIEW

File: config/slurm.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  user: "centos"
+  num_nodes: 1
+  root_volume_size: 0
+  node_resource: "Cluster::Node" 
+  extra_groups:
+    - "nfs_clients"
+    - "beegfs_clients"
+  nodenet_resource: "Cluster::NodeNet1WithPreallocatedFIP"
+  nodenet_fips:
+    - uuid: "8723c186-d15f-44d5-9421-e12bf2a03325"
```
Comment (brtkwr) [inline config/slurm.yml]: is there any reason we are specifying both uuid and ip other than ease? wouldn't it be nicer to retrieve the floating ip using the cli from the uuid instead?

**Review by brtkwr — APPROVED** [review]
lgtm

File: roles/requirements.yml [hunk 2]
```diff
@@ -7,8 +7,9 @@
 - src: https://github.com/stackhpc/ansible-role-cluster-infra
   version: newton-backport
   name: stackhpc.cluster-infra
-- src: https://github.com/cedadev/jasmin-cluster-infra-ansible
+- src: https://github.com/stackhpc/jasmin-cluster-infra-ansible
```
Comment (oneswig) [inline roles/requirements.yml]: The `issue-23` PR has been merged, before this commit gets merged can we update this?

**Review by oneswig — CHANGES_REQUESTED** [review]
Looks like there's a merge conflict with master to resolve - can the requirements.yml be updated when that gets dealt with.  Otherwise, looks good.

**Pull request merged**